### DISCUSSION TITLE
Zigbee2MQTT Alpine crashing occasionally.

### DISCUSSION THREAD

**Discussion by Scales82** [tteck/Proxmox#1568] [Help Needed]
Hi,
I just migrated to Z2M seperate from the onboard instance in HA. Installation worked fine, followed the instructions on the script no worries at all. But I'm noticing that all my Zigbee devices stop working after a period of time. I console into the LXC as root, cant see any problems and checked the log file at /var/log/messages and cant tell what is causing the crash. I have added the loglevel: debug to the configuration and still no clues as to what is going on. A reboot of the LXC and the problem goes away straight away.

I'd have uploaded log here but it has been overwritten now the LXC is running again. I've done nothing else to the container apart from setup my devices etc. Was working perfect on the HA instance. Pretty bummed out, I dont want to move back to that method.

**Comment by LeonArmston** [comment]
I have the same problem - every day I have to reboot it - multiple times a day.

**Comment by tteck** [comment]
I'd suggest moving to the Debian 12 install.

**Reply by LeonArmston** [reply]
Indeed - that's what I'll have to do. Was hoping avoid as last time not all devices re - paired automatically after the backup.

**Reply by LeonArmston** [reply]
Think my problem might be this [https://github.com/mercenaruss/uzg-firmware/issues/10](https://github.com/mercenaruss/uzg-firmware/issues/10) DHCP! Have changed to a static and might then change back from Debian to Alpine if so!

Seems my Zigbee POE coordinator (UZG-01) struggles to renew it's DHCP lease then causes Zigbee2mqtt to crash. A reboot of the UZG-01 then makes Zigbee2mqtt start again.
